client/tinychain/ml/mlr.py

```diff
@@ -0,0 +1,95 @@
+
+
+from tinychain.collection.tensor import einsum, Dense , Tensor
+from tinychain.ml import Layer, NeuralNet
+from tinychain.ref import After
+import tinychain as tc 
+import numpy as np
+
+
+class MultipleLinearRegression :
+    
+    @classmethod
+    def load (self,learningrate): 
+        return MultipleLinearRegression([learningrate])
+    
+    @property
+    def learningrate(self):  
```

Comment:
`MultipleLinearRegression` doesn't subclass any TinyChain native class (like `Tuple`) so you don't need to access properties this way